src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DefaultProjectSnapshotManager.cs

```diff
@@ -507,7 +580,7 @@ public override void ProjectRemoved(HostProject hostProject)
                 // We need to notify listeners about every project removal.
                 var oldSnapshot = entry.GetSnapshot();
                 _projects.Remove(hostProject.FilePath);
-                NotifyListeners(new ProjectChangeEventArgs(oldSnapshot, null, ProjectChangeKind.ProjectRemoved));
+                NotifyListeners(oldSnapshot, null, null, ProjectChangeKind.ProjectRemoved);
```

Comment:
I generally advocate for labeling null-parameters to help people looking at the calling code understand what's happening.

Reply:
Me too, I was lazy because there were already crimes committed here. Thank you for calling me on it :)